scalastyle-config.xml

```diff
@@ -18,59 +18,29 @@
   </parameters>
  </check>
  <check level="error" class="org.scalastyle.scalariform.EqualsHashCodeChecker" enabled="true"></check>
- <check level="error" class="org.scalastyle.scalariform.IllegalImportsChecker" enabled="true">
-  <parameters>
-   <parameter name="illegalImports"><![CDATA[sun._,java.awt._]]></parameter>
-  </parameters>
- </check>
  <check level="error" class="org.scalastyle.scalariform.ReturnChecker" enabled="true"></check>
- <!-- <check level="error" class="org.scalastyle.scalariform.NullChecker" enabled="true"></check> -->
  <check level="error" class="org.scalastyle.scalariform.CovariantEqualsChecker" enabled="true"></check>
  <check customId="console.io" level="error" class="org.scalastyle.file.RegexChecker" enabled="true">
   <customMessage>Avoid console i/o disable with // scalastyle:off console.io</customMessage>
   <parameters>
    <parameter name="regex"><![CDATA[println]]></parameter>
   </parameters>
  </check>
- <!-- <check level="error" class="org.scalastyle.scalariform.NumberOfTypesChecker" enabled="true">
```

Comment:
You're removing stuff that was already commented out, which is fine.  Is it different from setting `enabled="false"`? I guess you have to look at it when you open the file.